Assets/MRTK/SDK/Features/UX/Scripts/Pointers/SpherePointer.cs

```diff
@@ -31,7 +60,24 @@ public override SceneQueryType SceneQueryType
         /// <remarks>
         /// This creates a dead zone in which far interaction is disabled before objects become grabbable.
         /// </remarks>
-        public float NearObjectMargin => nearObjectMargin;
+        public float NearObjectMargin
+        {
+            get => nearObjectMargin;
+            set => nearObjectMargin = value;
+        }
+
+        [SerializeField]
+        [Min(0.0f)]
+        [Tooltip("Additional distance on top of sphere cast radius when pointer is considered 'near' an object and far interaction will turn off")]
```

Comment:
I think this comment is not correct, can you check it?

Reply:
Yeah I think the comment is accurate

Reply:
I think this tooltip is referring to nearObjectAxisLerp, which is a lerp amount, not a distance on top of the sphere cast radius. Could you double check again that it's accurate? I'm referring to this line: "Additional distance on top of sphere cast radius when pointer is considered 'near' an object and far interaction will turn off" when describing the variable "nearObjectAxisLerp"

Reply:
yeah I messed up and updated the comment